umap/static/umap/js/umap.core.js

```diff
@@ -73,8 +73,9 @@ L.Util.escapeHTML = (s) => {
   })
   return s
 }
-L.Util.toHTML = (r) => {
+L.Util.toHTML = (r, options) => {
   if (!r) return ''
+  const target = options && options.target || 'blank'
```

Comment:
Should we restrict the value to the [3 possible options](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target) only?

Reply:
Is that needed ? Given that user does not manually enter those values (they come from a radio input) ?

Reply:
Alright